commit 68fad751f7b4fc597c70a1e48cd2a4374cf74fbd
Author: Georg Sauthoff <mail@gms.tf>
Date:   Sat Jan 3 19:31:00 2026 +0100

    remove libnacl dependency declarations
    
    after release 2.22.0 elimnated code usage
    
    cf. https://asyncssh.readthedocs.io/en/latest/changes.html#release-2-22-0-21-dec-2025

diff --git a/pyproject.toml b/pyproject.toml
index d595baf..7b45ecd 100644
--- a/pyproject.toml
+++ b/pyproject.toml
@@ -36,7 +36,6 @@ dynamic = ['version']
 bcrypt    = ['bcrypt >= 3.1.3']
 fido2     = ['fido2 >= 2']
 gssapi    = ['gssapi >= 1.2.0']
-libnacl   = ['libnacl >= 1.4.2']
 pkcs11    = ['python-pkcs11 >= 0.7.0']
 pyOpenSSL = ['pyOpenSSL >= 23.0.0']
 pywin32   = ['pywin32 >= 227']
diff --git a/tox.ini b/tox.ini
index 03d2013..eaf7923 100644
--- a/tox.ini
+++ b/tox.ini
@@ -11,7 +11,6 @@ deps =
     aiofiles>=0.6.0
     bcrypt>=3.1.3
     fido2>=2
-    libnacl>=1.4.2
     pyOpenSSL>=23.0.0
     pytest>=7.0.1
     pytest-cov>=3.0.0
